packages/manager/tools/webpack-config/src/loaders/html-minifier.js

```diff
@@ -0,0 +1,18 @@
+const { minify } = require('html-minifier');
+// see : https://github.com/kangax/html-minifier
+
+module.exports = function translationsJson(source) {
+  if (this.mode !== 'production') {
+    return source;
+  }
+  return minify(source, {
+    continueOnParseError: true,
```

Comment:
Out of curiosity why do we need this ? On what cases is it failing and can't we fix HTML to be valid instead ?

Reply:
My reasoning was that minification plugin should not have the role of html validator. Since the scope of the tool is to optimize the bundle sizes. That's why i choosed the most permissive option, which uses the handle parse error behavior here : https://html.spec.whatwg.org/multipage/parsing.html#parse-errors. For syntax validation i prefer to rely on linting tools (which provide a realtime feedback during development). What do you think about it ?

Reply:
It makes sense